src/algorithms.jl

```diff
@@ -5246,6 +5246,28 @@ E. Hairer, S.P. Norsett, G. Wanner, (1993) Solving Ordinary Differential Equatio
 """
 struct Nystrom4 <: OrdinaryDiffEqPartitionedAlgorithm end
 
+"""
+    FineRKN5
+
+A 5th order explicit Runge-Kutta-Nyström method which can be applied directly on second order ODEs. Can only be used with fixed time steps.
+
+In case the ODE Problem is not dependent on the first derivative consider using 
+[`Nystrom5VelocityIndependent`](@ref) to increase performance.
```

Comment:
```suggestion
In case the ODE does not depend on the first derivative, consider using 
[`Nystrom5VelocityIndependent`](@ref) to increase performance.
```